core/server/master/src/main/java/alluxio/master/meta/MetaMasterSync.java

```diff
@@ -41,6 +41,8 @@
 @NotThreadSafe
 public final class MetaMasterSync implements HeartbeatExecutor {
   private static final Logger LOG = LoggerFactory.getLogger(MetaMasterSync.class);
+  /** We set a large retry day which means retry forever. */
+  private static final long RETRY_DAY = 100000;
```

Comment:
RETRY_DAY would mean to retry for a single day, I think this should be RETRY_DAYS